### PR TITLE
update client libraries to latest release version

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: src/main/webapp/download/index.jsp [hunk 1]
```diff
@@ -85,31 +85,31 @@
       </tr>
       <tr>
         <td>C#</td>
-        <td>3.0.0-beta1</td>
-        <td>2016-07-29</td>
+        <td>3.0.0-beta2</td>
+        <td>2016-08-04</td>
         <td><a href="http://selenium-release.storage.googleapis.com/3.0-beta1/selenium-dotnet-3.0.0-beta1.zip">Download</a></td>
```
Comment (lukeis) [inline src/main/webapp/download/index.jsp]: update the link here too :)

File: src/main/webapp/download/index.jsp [hunk 2]
```diff
@@ -85,31 +85,31 @@
...
+        <td>Selenium 3.0.0.b2</td>
+        <td>2016-08-03</td>
         <td><a href="http://pypi.python.org/pypi/selenium">Download</a></td>
         <td><a href="http://goo.gl/rHRdgk">Change log</a></td>
         <td><a href="http://goo.gl/sG1GfQ">API docs</a></td>
       </tr>
       <tr>
         <td>Javascript (Node)</td>
-        <td>3.0.0-beta-1</td>
+        <td>3.0.0-beta-2</td>
```
Comment (lukeis) [inline src/main/webapp/download/index.jsp]: node is still beta1

File: src/main/webapp/download/index.jsp (hunk 1 above)
Comment (manoj9788) [inline src/main/webapp/download/index.jsp]: Oh! yeah sure.

File: src/main/webapp/download/index.jsp (hunk 2 above)
Comment (manoj9788) [inline src/main/webapp/download/index.jsp]: Again I went by Changelog ..!! will update it.

**Pull request merged**